fuzzinator/job/validate_job.py

```diff
@@ -40,12 +40,17 @@ def validate(self):
 
         if issue:
             issue['test'] = self.issue['test']
+
+            self.ensure_id(issue)
+            assert 'id' in issue
+            assert 'id' in self.issue
```

Comment:
These asserts are not needed either, since `ensure_id` enforces to have an `id` field in issue.